### PR TITLE
#153 feat: add styles to map menu (accordion for hide description)

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx [hunk 1]
```diff
@@ -20,6 +22,21 @@ export const MapMenu = (props: MapMenuProps) => {
   const {markers, menuItem} = useContext(MapContext);
   const markerTypes = Array.from(new Set(markers.map(marker => marker.markerType)));
 
+  const [currentIdx, setCurrentIdx] = useState(-1);
```
Comment (SergioAJS) [inline gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx:25]: -1 is a magic number?
From the name, it is not clear what index is meant

File: gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx [hunk 2]
```diff
@@ -20,6 +22,21 @@ export const MapMenu = (props: MapMenuProps) => {
...
+  const [isOpen, setIsOpen] = useState(false);
```
Comment (SergioAJS) [inline gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx:26]: It is not clear from the name what is being opened

**Review by SergioAJS — CHANGES_REQUESTED** [review]

File: gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx (hunk 1 above)
Comment (Ekaterina1994) [reply gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx:25]: Create const defaultIndex and change naming to currentIndex

File: gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx [hunk 3]
```diff
@@ -20,6 +22,21 @@ export const MapMenu = (props: MapMenuProps) => {
...
+
+  const btnOnClick = (id: number) => {
```
[the remark this answers is not in the document]
Comment (Ekaterina1994) [reply gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx:28]: Change name to menuItemOnClick

File: gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply gf-ui-lib/src/components/Map/mapMenu/MapMenu.tsx:26]: Change to isMenuItemOpen

**Review by SergioAJS — APPROVED** [review]

**Review by scorpigg — APPROVED** [review]

**Pull request merged**